src/tests/datasets.h

```diff
@@ -4,74 +4,31 @@
 #include <string>
 #include <vector>
 
-static const auto test_data_dir = std::filesystem::current_path() / "input_data";
+#include "config/tabular_data/input_table_type.h"
+#include "parser/csv_parser/csv_parser.h"
 
-struct Dataset {
-    std::string name;
-    size_t hash;
+static auto const test_data_dir = std::filesystem::current_path() / "input_data";
+
+namespace tests {
+
+/// dataset configuration info to create an input table
+struct DatasetInfo {
```

Comment:
`DatasetInfo` name seems too general, let's rename it to something that indicates that it actually describes csv datasets

Reply:
Also, it's probably a good idea to rename this file accordingly